packages/server-wallet/e2e-test/e2e-utils.ts

```diff
@@ -56,4 +59,21 @@ export const waitForServerToStartAndResetDatabase = (
     }, pingInterval);
   });
 
+/**
+ * Asks the Pong Server to identify itself so that Ping may use this info in the creation
+ * of a new channel with Pong.
+ */
+export const getPongsParticipantInfo = async (pongServer: PongServer): Promise<Participant> => {
```

Comment:
This seems like a legitimate method that belongs on the `PingClient` -- the client would need to find out the peer's participant ID.